### PR TITLE
Adds hybrid licensing audit policy

### PR DESCRIPTION
**Pull request by kyleburnsdev**
Created sample to help organizations that can benefit from hybrid licensing detect if they have resources deployed that are not configured for this benefit

### CODE REVIEW

File: samples/SQL/audit-sql-db-without-hybrid-license/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+Generates audit record if SQL DB is not configured for hybrid license
+
+## Try on Portal
+
+[![Deploy to Azure](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#blade/Microsoft_Azure_Policy/CreatePolicyDefinitionBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-policy%2Fmaster%2Fsamples%2FSQL%2Faudit-sql-db-without-hybrid-license%2Fazurepolicy.json)
+
+## Try with PowerShell
+
+````powershell
+$definition = New-AzPolicyDefinition -Name "audit-sql-db-without-hybrid-license" -DisplayName "Audit SQL DB without Hybrid Licensing" -description "Generates audit record if SQL DB is not configured for hybrid license" -Policy 'https://portal.azure.com/#blade/Microsoft_Azure_Policy/CreatePolicyDefinitionBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-policy%2Fmaster%2Fsamples%2FSQL%2Faudit-sql-db-without-hybrid-license%2Fazurepolicy.rules.json' -Parameter 'https://portal.azure.com/#blade/Microsoft_Azure_Policy/CreatePolicyDefinitionBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-policy%2Fmaster%2Fsamples%2FSQL%2Faudit-sql-db-without-hybrid-license%2Fazurepolicy.parameters.json' -Mode All
```
Comment (pilor) [inline samples/SQL/audit-sql-db-without-hybrid-license/README.md]: >-Parameter 'https://portal.azure.com/#blade/Microsoft_Azure_Policy/CreatePolicyDefinitionBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-policy%2Fmaster%2Fsamples%2FSQL%2Faudit-sql-db-without-hybrid-license%2Fazurepolicy.parameters.json' [](start = 467, length = 255)

please remove -Parameter here and --params below

Comment (pilor) [inline samples/SQL/audit-sql-db-without-hybrid-license/README.md]: >-Mode All [](start = 723, length = 9)

mode Indexed please

Comment (pilor) [inline samples/SQL/audit-sql-db-without-hybrid-license/README.md]: >https://portal.azure.com/#blade/Microsoft_Azure_Policy/CreatePolicyDefinitionBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-policy%2Fmaster%2Fsamples%2FSQL%2Faudit-sql-db-without-hybrid-license%2Fazurepolicy.rules.json [](start = 228, length = 237)

these URLS for powershell/CLI should only be the raw.githubusercontent.com URL, not including the portal.azure.com prefix

Comment (kyleburnsdev) [reply samples/SQL/audit-sql-db-without-hybrid-license/README.md]: Done. Thanks for the catch. I had thought based on other samples that it was required to include the reference even if parameters JSON was an empty object

Comment (kyleburnsdev) [reply samples/SQL/audit-sql-db-without-hybrid-license/README.md]: Done

**Pull request merged**